apps/frontend/src/libs/components/button/styles.module.css

```diff
@@ -15,11 +14,8 @@
 	outline: 0;
 }
 
-.fluid {
-	width: 100%;
-}
-
 .primary {
+	line-height: 24px;
```

Comment:
@fshabanov On the project we have to use values ​​in px or like line-height: 1.5 ??

Reply:
I'd prefer using unitless values for line-height (meaning `1.5` or `%`)  - it's more flexible when the line-height depends on the font size. Was there a specific reason why we used `px` here, @marharita08 ?

Reply:
I used `px` because this units specified in figma. But if you think that unitless value is better, I will update it.